### DISCUSSION TITLE
Unable to verify

### DISCUSSION THREAD

**Discussion by Atlantiscitizens** [hashlips-lab/nft-erc721-collection#244] [General]
Hi guys, I'm trying to verify my contract but I keep getting this error message. I haven't changed anything to the coding and can't work out what is going wrong. It worked at first and then it suddenly stopped working.

As said I did not change the coding so im using solidity as high as possible. 

The contract address I keep getting when I'm deploying is: 0x5FbDB2315678afecb367f032d93F642f64180aa3. 
If I look it up on etherscan I see the contract there, but that is definitely not mine... it has like 42 eth balance, which I definitely not have.

Is it possible something got glitched or hacked that my contract keeps deploying to this address? Or maybe something else?

$ hardhat verify --constructor-args config/ContractArguments.ts 0x5FbDB2315678afecb367f032d93F642f64180aa3 --network testnet
Error in plugin @nomiclabs/hardhat-etherscan: The contract you want to verify was compiled with solidity 0.7.3, but your configured compiler version is: 0.8.9.

Possible causes are:
  - You are not in the same commit that was used to deploy the contract.
  - Wrong compiler version selected in hardhat config.
  - The given address is wrong.
  - The selected network (testnet) is wrong.

**Comment by joven1555** [comment]
You selected Rinkeby network on Etherscan, right?

**Reply by Atlantiscitizens** [reply]
Yeah I did
